packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { MainFeedPageProps } from '../MainFeedPage';
+import { PlusHeader } from './PlusHeader';
+
+export default function PlusLayout({
+  children,
+}: MainFeedPageProps): ReactElement {
+  return (
+    <main className="flex h-screen flex-col bg-blur-glass before:absolute before:inset-0 before:-z-1 before:bg-[url('https://daily-now-res.cloudinary.com/image/upload/s--q2q8AL3Q--/f_auto/v1731397654/Upgrade_to_plus_i8phhr')] before:content-['']">
```

Comment:
Use absolute img? And then move image to cloudinary constants?

Reply:
I was too lazy to move the flex order around 😂
We can do it yeah, probably should.

Also the image is not correct now I saw.

Reply:
We could always add the background images to tailwind 😬